### DISCUSSION TITLE
Osu!lazer crashing when opening the "browse" tab

### DISCUSSION THREAD

**Discussion by ShidoSetsuna** [ppy/osu#16305] [Q&A]
Some things to note:
Operating system: Win 10
Osu lazer version: 25 of December 2021 (latest as of writing this)

As the title says, when I click on the beatmap browsing tab, Osu lazer completely freezes. 
And after a short while the "This program is not responding" message appears. I also want to mention that
this has never happened before, and happened right after updating to the latest version. Also, I wasn't logged in 
when this happened, but have tried logging in and yes, logging in does fix it. So the crash is only happening when im not logged in.

**Comment by peppy** [comment]
Please [provide logs](https://github.com/ppy/osu/discussions/15188) to help us understand your issue.
